### PR TITLE
Updating docs new attribs

### PR DESCRIPTION
**Pull request by olaaustine**
[ENSVAR-5285](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-5285)
[ENSVAR-5289](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-5289)

### CODE REVIEW

File: public/gene2phenotype/README [hunk 1]
```diff
@@ -24,7 +24,20 @@ consequence. The header fields are:
...
+  - comments                      Public comments made by our curator 
+  - variant consequence           semi-colon separated list of possible variant consequence attributes :
+                                  splice_region_variant, splice_acceptor_variant, splice_donor_variant, start_lost, frameshift_variant,stop_gained, stop_lost, missense_variant, 
+                                  inframe_deletion, 5_prime_UTR_variant, 3_prime_UTR_variant,synonymous_variant, intron_variant, regulatory_region_variant, intergenic_variant, 
+                                  inframe_insertion, NMD_triggering,
+                                  NMD_escaping, stop_gained_NMD_triggering, stop_gained_NMD_escaping,splice_donor_variant_NMD_triggering, 
+                                  frameshift_variant_NMD_triggering,splice_acceptor_variant_NMD_escaping, splice_acceptor_variant_NMD_triggering, splice_donor_variant_NMD_escaping, 
+                                  frameshift_variant_NMD_escaping, loss_of_function_variant, gain_of_function_variant
+  - disease ontology              Disease ontology accession
```
Comment (sarahhunt) [inline public/gene2phenotype/README:38]: There is Disease Ontology with DO:* accessions, so this is confusing  - the description 'MONDO accession' would be clearer.

**Comment by olaaustine** [comment]
Do not merge until after https://github.com/EBI-G2P/ensembl-gene2phenotype/pull/76 has been merged. 
Thank you

**Comment by dglemos** [comment]
https://github.com/EBI-G2P/ensembl-gene2phenotype/pull/76 is merged.

**Review by dglemos — APPROVED** [review]

**Comment by dglemos** [comment]
Merged into release/108 and master

**Pull request merged**